holidayextras/ui-toolkit
# UXUI-78 - Adding styling for inputs

#### What does this PR do? (please provide any background)
- We have a few PR's in now to add or use input into the toolkit. These have been created but don't look particularly great.

#### What tests does this PR have?
None

#### How can be this tested?
- Run the UI toolkit 
- `grunt docs` is your best bet, edit the HTML in dev tools and add an input to the page `<input type="text" />` for example
- check ho it looks - not great
- add `class="component-input"` to the input
 It should no look good.

#### Screenshots / Screencast
![Input](https://cloud.githubusercontent.com/assets/778942/8480495/f32b630a-20d5-11e5-8440-3ff9e8af4749.png)

#### What gif best describes how you feel about this work?
![Input](http://i.imgur.com/TYIPIY3.gif)
(it makes me sad that a lot of HX developers wont understand this GIF)
---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
